chemistrylab/extract_algorithms/extractions/water_oil_v1.py

```diff
@@ -112,7 +114,7 @@ def reset(self, extraction_vessel):
 
         oil_vessel_material_dict = {}
         C6H14 = material.C6H14
```

Comment:
Would it make more sense to make this an object rather than doing it multiple times later?